src/main/kotlin/com/mtereshchuk/account/controller/RequestValidator.kt

```diff
@@ -0,0 +1,48 @@
+package com.mtereshchuk.account.controller
+
+import com.mtereshchuk.account.model.*
+import java.math.BigDecimal
+
+/**
+ * @author mtereshchuk
+ */
+typealias ValidationError = String
+
+class RequestValidator {
```

Comment:
Usually, I prefer to use validation annotations like `@Min` or `@Size` as in JAX-RS, which allows add contracts for REST object creation to guarantee that an invalid instance wouldn't be created, but here for simplicity the simple REST object was used